### PR TITLE
Fix scrollbar apperance

### LINKED ISSUE #34064: [NC25] Scrollbar of main section is not of style thin
Observed in Firefox

grafik

Possibly `main` has to be added?
https://github.com/nextcloud/server/blob/a09077cd178e309099e0bd8109eaf2d91fc77a5c/core/css/styles.css#L38-L49

### PR DESCRIPTION
**Pull request by dartcafe**
fix #34064 

Before
grafik
After
grafik

related #33736

### CODE REVIEW

**Comment by skjnldsv** [comment]
Screenshots please? :)

File: core/css/styles.css [hunk 1]
```diff
@@ -35,7 +35,7 @@
  * @license GNU AGPL version 3 or any later version
  *
  */
-html, body, div, span, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, code, del, dfn, em, img, q, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, dialog, figure, footer, header, hgroup, nav, section {
+html, body, div, span, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, code, del, dfn, em, img, q, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, dialog, figure, footer, header, hgroup, nav, section, main {
```
Comment (szaimen) [inline core/css/styles.css:38]: Needs to be done in styles.scss and the compiled and comitted...

Comment (dartcafe) [reply core/css/styles.css:38]: `npm run sass` is throwing an error.
```
Directory "core/css/" may not be a positional arg.
To compile all CSS in "core/css/" to "apps/*/css", use `sass core/css/:apps/*/css`.
```
manually compiled with `sass --load-path core/css core/css/ apps/*/css`

**Review by jancborchardt — APPROVED** [review]
Good catch! :)

**Review by PVince81 — APPROVED** [review]
:+1:

**Pull request merged**